core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties

```diff
@@ -0,0 +1,33 @@
+title=Manage Legacy API Token usage
+contextMessage=The following users are using Legacy API Token. \
+As that old system stores the credentials in a recoverable format, it is strongly advised to migrate to the new system.<br /> \
+In previous versions of Jenkins, the creation of a new user implied to the generation of an Legacy API Token. \
+Often majority of users are not using that token and so provide a larger potential attack surface than required.<br /> \
+From now, the automatic generation of token is determined by configuration.
+recommendationMessage=For such reasons, we recommend to: \
+<ul>\
+  <li>revoke the tokens that were never used and</li>\
+  <li>ask users that are effectively using their legacy token to generate a new token using the new system and revoking the legacy one</li>\
+</ul>
+UserId=User Id
+UserFullName=User full name
+TokenName=Token name
+NumDaysSinceCreation=Days since creation
+NumOfUse=# of use
+NumDaysSinceLastUse=Days since last use
+HasFreshToken=Fresh token?
+HasFreshToken_tooltip=A fresh token is a token from the new system \
+that was created after the last use of the legacy token. \n\n\
+Also it is important to notice that legacy token's creation date is reset \n\
+every time the instance is restarted if they were never saved before. So it was never used, the date is not really accurate.
+HasMoreRecentlyUsedToken=Recent token?
+HasMoreRecentlyUsedToken_tooltip=A recently used token is a token from the new system \
```

Comment:
nit: I might reword this to "A recently used token is one that was created using the new system and used more recently than the user's legacy token. Note: the creation date of legacy tokens which have never been used are reset every time Jenkins is restarted, which means that the date may be inaccurate."